include/seqan3/io/record.hpp

```diff
@@ -338,31 +338,31 @@ namespace seqan3
 
 //!\brief Free function get() for seqan3::record based on seqan3::field.
 template <field f, typename field_types, typename field_ids>
-auto & get(record<field_types, field_ids> & r)
+auto & get SEQAN3_DEPRECATED_310 (record<field_types, field_ids> & r)
```

Comment:
In `include/seqan3/range/container/concatenated_sequences.hpp` we wrote the SEQAN3_DEPRECATED_310 tag before the function (I believe in other places we did the same).
```suggestion
SEQAN3_DEPRECATED_310 auto & get(record<field_types, field_ids> & r)
```